src/LibDeclarations/NativeExtensions/FeaturePoints.md

```diff
@@ -0,0 +1,61 @@
+# Feature point cloud

```

Comment:
Is there a better place for this file to live?

Reply:
There is not, but I don't think that's necessary.  From the BabylonNative perspective we are just fulfilling the extended WebXR contract which might differ between supported devices.  From a feature perspective this doesn't really differ from other features on the BabylonNative side in any meaningful way.